CarRR-kjar/src/main/resources/categorisation_tagging_rules/categorisation_rules.drl

```diff
@@ -58,10 +58,9 @@ no-loop
         $vehicle: Vehicle(fuelConsumption < 14, fuelConsumption > 10, length > 430, height > 170, width > 195, cargoVolume + passengerAreaVolume > 3300, seatsNo >= 5, doorNo >= 4, pricePerDay > 300, $features: features)
         Number(intValue < 2) from accumulate(
                     ExtraFeatures(
+                    	$id: id
                     ) from $features,
-                    init(int count = 0;),
-                    action(count += 1;),
-                    result(count)
+                    sum($id)
```

Comment:
Are you sure that sum counts the number of occurrences instead of actually summing up the value of id's?

Reply:
Dzizs krajst